.github/workflows/platform.ManageIssueForFailingPipelines.yml

```
@@ -0,0 +1,31 @@
+name: 'create/update/close issues for failing pipelines in main'
+
+on:
+  schedule:
+    - cron: '30 5 * * *'
+jobs:
+  manage-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - env:
+          GITHUB_TOKEN: ${{ secrets.PLATFORM_PROJECT_TOKEN }}
+        name: Manage issues
+        shell: pwsh
+        run: |
+          # Load used functions
+          . (Join-Path $env:GITHUB_WORKSPACE 'utilities' 'pipelines' 'manageIssueForFailingPipelines' 'New-IssuesForFailingPipelines.ps1')
+
+          $functionInput = @{
+            repo = "${{ github.repository_owner }}/${{ github.event.repository.name }}"
+            limitNumberOfRuns = 500
+            limitInDays = 2
+          }
+
+          Write-Verbose "Invoke task with" -Verbose
+          Write-Verbose ($functionInput | ConvertTo-Json | Out-String) -Verbose
+
+          New-IssuesForFailingPipelines @functionInput -Verbose # -WhatIf
```

```suggestion
          Set-IssueForWorkflow @functionInput -Verbose
```
It is a best practice that PowerShell functions are singular (as they should pretend to operate on at least one item). And further, I'd suggest to use `Set` instead of new, as we also update (i.e. set) also existing issues and not always create new.

Note: This name change must be updated everywhere (file + references)

---

done